src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -675,6 +675,38 @@ private SubscriptionGeneralDataAndStats GetSubscriptionInternal(SubscriptionStat
             return subscriptionData;
         }
 
+        public int WaitForClusterStabilizationInMs;
+
+        public bool ShouldWaitForClusterStabilization
```

Comment:
why this is not a method? It looks a bit too complex

Reply:
done